src/main/java/com/back/domain/member/member/service/MemberService.kt

```
@@ -134,43 +129,40 @@ class MemberService(
-            if (!contentType.startsWith("image/")) throw ServiceException(400, "이미지 파일만 업로드 가능합니다.")
+        image?.takeIf { !it.isEmpty }?.let {
+            val contentType = it.contentType ?: ""
+            if (!contentType.startsWith("image/")) {
+                throw ServiceException(400, "이미지 파일만 업로드 가능합니다.")
+            }
+
             try {
-                val imageUrl = s3Service.upload(image, "member/${info.id}/profile")
+                val imageUrl = s3Service.upload(it, "member/${info.id}/profile")
                 info.updateImageUrl(imageUrl)
             } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)